### ISSUE TITLE
Unstable solutions not dotted

### ISSUE THREAD

**Issue by tt386** [pyrates-neuroscience/PyCoBi#2]
Hello, 

I have cloned the master branch of the git repo, and using the `qif_sfa.py` example script. I notice that the unstable solutions between the Hopf bifurcations are solid, whereas in the [paper](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1011761) it's shown to be properly dotted. Curiously, the subsequent plot shows that dotted lines are possible, just still not between the Hopf bifurcations.
![Fig2](https://github.com/user-attachments/assets/768ed66b-f412-43c6-a37b-999c4d773e4b)
![Fig3](https://github.com/user-attachments/assets/5b4018ef-d9e1-40bf-aced-f694b7ddcf8e)

As I mentioned in a previous post, my versions are (from the cloned repo):
PyCoBi: 0.8.7
PyRates: 1.0.6

**Comment by Richert** [comment]
Hi Thorsten,

thank you very much for bringing this to my attention. This is a bug and I have identified what caused it. It was a problem with automatically merging Auto-07p solution branches. That led to wrong solution indices in PyCoBi, causing it to pull up the stability indicators for the wrong solutions. I must have caused this bug at some point, when I changed how solutions are merged in PyCoBi. I have just implemented and pushed a fix to the master branch. Will include that in a new release of PyCoBi later today, probably.

Let me know if those changes solve the problem for you as well, then I will close this issue.
